### PR TITLE
Update commands ssh, tunnel to support orgs

### PR DESCRIPTION
**Pull request by srlowe**
Change-type: patch
Connects-to: #2119
Signed-off-by: Scott Lowe <scott@balena.io>

Resolves: #   
Change-type: major|minor|patch   
Depends-on: <url>   
See: <url>   

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

File: lib/utils/ssh.ts [hunk 1]
```diff
@@ -162,6 +162,7 @@ function sshErrorMessage(exitSignal?: string, exitCode?: number) {
 				msg.push(`
 Are the SSH keys correctly configured in balenaCloud? See:
 https://www.balena.io/docs/learn/manage/ssh-access/#add-an-ssh-key-to-balenacloud`);
+				msg.push('Are you using `sudo` by mistake?');
```
Comment (pdcastro) [inline lib/utils/ssh.ts:165]: `sudo` doesn't usually change the homedir / `$HOME` env var, right? -
```
 $ /bin/bash -c 'echo $HOME'
/Users/paulo

$ sudo /bin/bash -c 'echo $HOME'
Password:
/Users/paulo
```
But logging in as `root` does:
```
$ sudo su -
Paulos-MBP:~ root# echo $HOME
/var/root
```
So perhaps:
```
msg.push('Are you accidentally running under the `root` user account?');
```

**Review by pdcastro — COMMENTED** [review]
Nice work 👍

**Comment by pdcastro** [comment]
Tests are failing on Alpine with the following error, I don't remember seeing it before... 😬 
```
  1) LivepushManager::setupFilesystemWatcher
       for project no-docker-compose/basic
         should trigger change events for paths that are not ignored:
     Error: ENOSPC: System limit for number of file watchers reached, watch '/tmp/build/01326af6/versioned-source/tests/test-data/projects/no-docker-compose/basic'
      at FSWatcher.start (internal/fs/watchers.js:169:26)
      at Object.watch (fs.js:1339:11)
      at createFsWatchInstance (node_modules/chokidar/lib/nodefs-handler.js:119:15)
      at setFsWatchListener (node_modules/chokidar/lib/nodefs-handler.js:166:15)
      at NodeFsHandler._watchWithNodeFs (node_modules/chokidar/lib/nodefs-handler.js:331:14)
      at NodeFsHandler._handleDir (node_modules/chokidar/lib/nodefs-handler.js:559:19)
      at runMicrotasks (<anonymous>:null:null)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
      at NodeFsHandler._addToNodeFs (node_modules/chokidar/lib/nodefs-handler.js:608:16)
      at /tmp/build/01326af6/versioned-source/node_modules/chokidar/index.js:445:21
      at Function.all (<anonymous>:null:null)
```

File: lib/utils/ssh.ts (hunk 1 above)
Comment (srlowe) [reply lib/utils/ssh.ts:165]: (I like `accidentally` vs `by mistake` though)

**Review by pdcastro — APPROVED** [review]

**Pull request merged**